server/routes/request.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const {
+  getRequests,
+  makeRequest,
+  updateRequest,
+} = require("../controllers/request");
+const protect = require("../middleware/auth");
+const router = express.Router();
+
+router.route("/").get(protect, getRequests);
+router.route("/").post(protect, makeRequest);
+router.route("/:id").put(protect, updateRequest);
```

Comment:
```suggestion
router.route("/:id").patch(protect, updateRequest);
```
If we are only updating part of the request doc, the PATCH HTTP verb seems more appropriate